### ISSUE TITLE
Mentor FAQ: Add section about how intern selections are resolved

### ISSUE THREAD

**Issue by sagesharp** [outreachy/website#362]
Add a section to the mentor FAQ about how to resolve an intern selection conflict:

In the case where applicants are selected for multiple projects, we go through a few steps:

1. Start a discussion with the mentors about the quality of the applicant's contributions.

Mentors - Has the applicant shown they have the skills to be a successful intern? Are their contributions of high quality? Have they been consistent about communicating with mentors? Have they been consistently contributing?

2. Does one project have another strong applicant they want to accept?

If one project has no other strong applicants, the preference is that applicant interns with that project. The goal is to have as many interns as possible.

3. What is the source of the intern funding for both projects?

There is a slight preference from the Outreachy organizers to have the intern accepted by a community who is funding them from community sponsors. This maximized the number of interns we can accept.

4. What is the intern's preference?

If both mentors still want to accept the applicant after the discussion, the Outreachy organizer should contact the applicant and ask them which project they want to work on.

**Comment by gconnect** [comment]
Hi, @sagesharp I would like to work on this. I believe the content to be added to the Mentor FAQ are what you already provided above in the issue description?

**Comment by sagesharp** [comment]
@gconnect Thanks for the offer of help! For looking at future issues to solve, I would recommend you take a look at issues marked '[help-wanted](https://github.com/outreachy/website/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)'. Those issues are a little more well-defined than some of the issues I file as notes to myself. If you're new to Django, please look at the '[newcomers-welcome](https://github.com/outreachy/website/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+label%3Anewcomers-welcome)] issues.

Unfortunately, this issue isn't an easily solvable issue. The mentor FAQ is currently stored in the website's CRM database. It isn't a template that's currently in this repository. You would have to first create a new template for the mentor FAQ page, copy the information that's on the [current mentor FAQ page](https://www.outreachy.org/mentor/mentor-faq/), add a new view for the page in home/views.py, add the URL to the ones listed in home/urls.py, and then finally add the new information about intern selection conflicts.

If you're up for that challenge, you can take a look at how the opportunities page was moved from the CRM database to the website Django repository. Using view page source in your browser on the mentor FAQ page may help. [This pull request](https://github.com/outreachy/website/pull/321/commits/2add76b3b7d60a466e3411bf08310e0af6748790) would help you understand the pieces you'd need to put in place to move the mentor FAQ page into this repository.

**Comment by gconnect** [comment]
Thanks @sagesharp  for the detailed explaination. Its a lot clearer now. I am up for the challenge. I will give it a shot.

**Comment by sagesharp** [comment]
@gconnect Great to hear you're up for a challenge! Do let me know if you run into problems with the installation instructions.

**Comment by gconnect** [comment]
I am having this error while trying to do ./manage.py migrate
![image](https://user-images.githubusercontent.com/23031920/74111447-bf015500-4b94-11ea-8ef0-5fba6e21ee30.png)

**Comment by sagesharp** [comment]
@gconnect Could you copy-paste all the commands you've run so far, starting from the 'git clone' command?

I notice you're running python 3.8. The Outreachy website is currently using Python 3.6.2. That could be part of the problem. Pip should have warned you about using a newer version of Python. If you're following the installation instructions correctly (which we'll see from which commands you've run) than trying to install an older version of Python might help.

**Comment by gconnect** [comment]
These are the commands I have ran so far
git clone https://github.com/outreachy/website.git
cd website
pipenv install
pipenv shell
npm install
./manage.py migrate (is at this point am having the error)

**Comment by sagesharp** [comment]
Hi @gconnect. Debian doesn't have Python 2.8 available yet, so I'm unable to update my own development environment to the same Python version as yours. I looked over your list of commands, and they do look correct.

I suspect the issue is something new with Python 2.8. Could you try installing Python 2.7 and see if that fixes your issue? Otherwise I'll wait until Debian updates the Python package and see if I run into the same errors.

**Comment by gconnect** [comment]
I guess you meant to say Python 3.7?

**Comment by sagesharp** [comment]
Yes, sorry, Python 3.7. I'm currently running Python 3.7.6 on my machine.

**Comment by sagesharp** [comment]
@gconnect Jamey recently fixed a Python 3.8 incompatibility in our Django project. You should now be able to get your website development environment set up using Python 3.8. I'm so sorry it took so long to fix this issue. We'd love if you want to pick up this issue again, or any other issue tagged "newcomers welcome" or "help wanted".

**Comment by gconnect** [comment]
Thanks. I will look at the issue again. And get to work.
